scripts/removeDarkmode.js

```
@@ -0,0 +1,91 @@
+  const { filePath, patterns } = configFile;
+  const contentFile = JSON.parse(fs.readFileSync(filePath, "utf8"));
+  patterns.forEach((pattern) => deleteNestedProperty(contentFile, pattern));
+  fs.writeFileSync(filePath, JSON.stringify(contentFile));
+}
+
+function deleteNestedProperty(obj, propertyPath) {
+  const properties = propertyPath.split(".");
+  let currentObj = obj;
+  for (let i = 0; i < properties.length - 1; i++) {
+    const property = properties[i];
+    if (currentObj.hasOwnProperty(property)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)